react/homework/src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
Look like the code is not update for Fibonacci home work, as this is default reach app code.